Include 'oc set deployment-hook' and 'oc set build-hook' in doc

csrwng opened the pull request:
Adds CLI instructions for deployment and build hooks.

csrwng commented:
@bparees ptal

bparees commented:
lgtm, @openshift/team-documentation ptal

ahardin-rh commented on dev_guide/builds.adoc:
```diff
@@ -1734,6 +1734,26 @@ Providing both `*script*` and `*command*` simultaneously creates an invalid
 build hook.
 ====
 
+==== Using the command line
```
Title case for the full heading

ahardin-rh replied:
Please add the following anchor above the title:
[[builds-using-the-command-line]]

ahardin-rh commented on dev_guide/builds.adoc:
```diff
@@ -1734,6 +1734,26 @@ Providing both `*script*` and `*command*` simultaneously creates an invalid
 
+==== Using the command line
+
+The `*oc set build-hook*` command can be used to set the build hook for a build config.
```
s/ `*oc set build-hook*`/`oc set build-hook`
s/config/configuration

ahardin-rh commented on dev_guide/deployments.adoc:
```diff
@@ -552,6 +552,19 @@ pod will have the following properties:
 * The hook failure policy is `Abort`, meaning the deployment will fail if the hook fails.
 * The hook pod will inherit the `data` volume from the deployment configuration pod.
 
+==== Using the command line
```
Title case for the full heading

ahardin-rh replied:
Please add the following anchor above the title:
[[deployments-using-the-command-line]]

ahardin-rh commented on dev_guide/deployments.adoc:
```diff
@@ -552,6 +552,19 @@ pod will have the following properties:
+==== Using the command line
+
+The `*oc set deployment-hook*` command can be used to set the deployment hook for a
+deployment config. For the example above, you can set the pre-deployment hook with the
```
No asterisks around oc set build-hook, just back ticks
s/config/configuration

ahardin-rh commented:
@csrwng @bparees just a few comments in line. Please confirm that my labels and milestone are correct. Thanks!

csrwng commented:
@ahardin-rh thx for the quick review. I've addressed your comments.

For the labels, this is meant to go in the next release of OpenShift, so post-3.2 and it does apply to all of them (dedicated, enterprise, and online).

ahardin-rh commented:
@csrwng Thanks! Looks good! Also, thanks for your guidance on the labels.

ahardin-rh commented:
[rev_history]
|link:../dev_guide/builds.html[Builds]
|Added information about the link:../dev_guide/builds.html#build-hooks-using-the-command-line[`oc set build-hook` command].
%
|link:../dev_guide/deployments.html[Deployments]
|Added information about the link:../dev_guide/deployments.html#deployment-hooks-using-the-command-line[`oc set deployment-hook` command].
%
